station_harvest/code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -0,0 +1,10 @@
+/datum/reagent/fuel/oil/expose_turf(turf/exposed_turf, reac_volume)//splash the fuel all over the place
```

Comment:
I changed this, it was previously on base ``/datum/reagent/fuel`` which I assumed to be accidental because it already has one, and this is drone blood.